pytube/__main__.py

```diff
@@ -117,6 +118,14 @@ def init(self):
                 self.watch_html,
             )['args']
 
+        # load the player_response object (contains subtitle information)
+        apply_mixin(self.player_config_args, 'player_response', json.loads)
+
+        player_response = self.player_config_args['player_response']
+        if ('playabilityStatus' in player_response and
```

Comment:
What does it mean to not have `playabilityStatus`?

Reply:
not sure, but it sometimes this key is not in the `dict`